src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -134,6 +187,13 @@ private void canvasRestApiCall(HttpServletRequest request,
 		}
 
 	}
+	
+	private void esbRestApiCall(HttpServletRequest request,
```

Comment:
Both the direct to canvas and esb to canvas calls get the same canvas data.  It may be clearer to name them consistently after the channel they use instead of naming one for the source (canvas) and one for the channel (esb).